server/Makefile

```
@@ -48,6 +52,10 @@ fmt:
 test:
 	$(GO) test ./...
 
+.PHONY: test-with-coverage
+test-with-coverage:
```

do you think depending on `generate` here may be helpful for test targets?

```suggestion
test-with-coverage: generate
```

---

Yes, since we don't want tests running against stale mocks.  I wrote a `mockgen` wrapper script and threw it under bin/ so that `go generate` picks up the tool pinning I did for it.

Fixed in 212c7a1 and 00f6b95.